### PR TITLE
Switch PWM ports and joystick configuration for new kitbot

### PR DESCRIPTION
**Pull request by ErikBoesen**

### CODE REVIEW

File: robot/robot.py [hunk 1]
```diff
@@ -23,10 +23,14 @@ def createObjects(self):
         #   Dig | 0/1
         #   2^1 | Left/Right
         #   2^0 | Front/Rear
-        self.lf_motor = wpilib.Victor(0b00)  # => 0
-        self.lr_motor = wpilib.Victor(0b01)  # => 1
-        self.rf_motor = wpilib.Victor(0b10)  # => 2
-        self.rr_motor = wpilib.Victor(0b11)  # => 3
+        #self.lf_motor = wpilib.Victor(0b00)  # => 0
```
Comment (ArchdukeTim) [inline robot/robot.py:26]: I don't even need to ask why you're using binary. But I will anyway. Why?

**Review by ArchdukeTim — APPROVED** [review]

File: robot/robot.py (hunk 1 above)
Comment (ErikBoesen) [reply robot/robot.py:26]: Because when it's represented that way it aligns with the left/right/front/back layout.

Comment (twinters007) [reply robot/robot.py]: How so?

Comment (ArchdukeTim) [reply robot/robot.py:26]: Ah. Neat trick

Comment (ErikBoesen) [reply robot/robot.py:26]: Code is poetry, Tim!

**Pull request merged**